content/en/docs/tutorials/stateful-application/basic-stateful-set.md

```diff
@@ -131,6 +131,11 @@ Notice that the `web-1` Pod is not launched until the `web-0` Pod is
 _Running_ (see [Pod Phase](/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase))
 and _Ready_ (see `type` in [Pod Conditions](/docs/concepts/workloads/pods/pod-lifecycle/#pod-conditions)).
 
+#### Custom replica numbering
+
+Note: To customize the replica numbering of pods in a StatefulSet, see
+[Ordinals](docs/tutorials/stateful-application/basic-stateful-set/#ordinals).
```

Comment:
We usually avoid H4 in the document. Maybe render this as a `note` shortcode would be better?

```suggestion
{{< note >}}
To customize the replica numbering of pods in a StatefulSet, see
[Ordinals](/docs/concepts/workloads/controllers/statefulset/#ordinals).
{{< /note >}}
```